### ISSUE TITLE
a graduate student

### ISSUE THREAD

**Issue by demokiss** [forallx94/log_anomaly_detection#1]
Hello, I have some questions about the code of Unsupervised Log Message Anomaly Detection you reproduced: why are the accuracy and recall very low when testing against BGL data set?

**Comment by forallx94** [comment]
For time reasons, I didn't dig into it.

This is just idea
1. It may be a difference between the dataloader method of BGL and HDFS.
	In HDFS, divide by blk_id, but in BGL, devide by 1 minite
2. It may be a difference between the labeling method of BGL and HDFS.
	In HDFS, label by blk_id, in BGL, label by each state
3. It may be because don't have a hyperparameter tuning

I create these three hypotheses but have not tested them.

**Comment by demokiss** [comment]
Thank you very much for your answer. I will try to test it. Now I use the data division method of the original paper, but I find it can't reach the result.

**Comment by forallx94** [comment]
I didn't get the original author's code. It may be different because it is made by reading and referring to original paper.

**Comment by demokiss** [comment]
Thank you again for your answer. I'm very glad to communicate with you. I hope we can keep in touch if there are any questions.

**Comment by demokiss** [comment]
Hello, I'm sorry to trouble you again. I see that your code reserved HDFS dataset, but you only used 100K, could you share all of them?

**Comment by forallx94** [comment]
Github loghub : https://github.com/logpai/loghub
Download link : https://zenodo.org/record/3227177
